static/app/views/alerts/rules/issue/index.tsx

```diff
@@ -405,6 +411,36 @@ class IssueRuleEditor extends AsyncView<Props, State> {
     this.fetchPreview(true);
   }, 1000);
 
+  // As more incompatible combinations are added, we will need a more generic way to check for incompatibility.
+  checkIncompatibleRule = debounce(() => {
+    const {rule} = this.state;
+    if (
+      !rule ||
+      !this.props.organization.features.includes('issue-alert-incompatible-rules')
+    ) {
+      return;
+    }
+
+    const conditions = rule.conditions;
+    let firstSeen = 0;
+    let regression = 0;
+    let reappearance = 0;
```

Comment:
could use `conditions.some(condition => condition.id.endsWith())` instead of a for loop

Reply:
ah wait you're showing which condition is incompatible